docs/docs/query-behind-the-scenes.md

```diff
@@ -2,11 +2,13 @@
 title: How Queries Work
 ---
 
-As part of Gatsby's data layer, GraphQL queries can be specified as tagged graphql expressions at the bottom of your component source file (e.g. [query for Gatsby frontpage](https://github.com/gatsbyjs/gatsby/blob/master/www/src/pages/index.js#L142)), StaticQueries within your components (e.g. [showcase site details](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/showcase-details.js#L96)), or fragments created by plugins (e.g. [gatsby-source-contentful](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-contentful/src/fragments.js)).
+In Gatsby, GraphQL queries are specified as tagged `graphql` expressions. These can be exported in your page source files, used in the `StaticQuery` component or used in the `useStaticQuery` hook in your React code. Plugins can also define fragments for use in queries.
 
-Note that this process only applies to queries that are specified directly in components or templates. It doesn't apply to queries involved in the creation of dynamic pages through your site's `gatsby-node.js` file (e.g. on [Gatsby's website](https://github.com/gatsbyjs/gatsby/blob/master/www/src/utils/node/docs.js#L42)).
+Note that the process outlined in this section only applies to queries that are specified in components or templates. It does _not_ apply to queries specificed in a `gatsby-node.js` file which are typically used for the creation of dynamic pages.
```

Comment:
Wonder if instead of completely removing example we could maybe link to https://www.gatsbyjs.org/tutorial/part-seven/#creating-pages?